cardano-node-chairman/src/Chairman/Process.hs

```diff
@@ -120,6 +128,28 @@ waitForProcess :: HasCallStack
 waitForProcess hProcess = GHC.withFrozenCallStack $ do
   H.evalM . liftIO $ catch (fmap Just (IO.waitForProcess hProcess)) $ \(_ :: AsyncCancelled) -> return Nothing
 
+procDist
+  :: String
+  -- ^ Package name
+  -> [String]
+  -- ^ Arguments to the CLI command
+  -> Integration CreateProcess
+  -- ^ Captured stdout
+procDist pkg arguments = do
+  contents <- liftIO $ LBS.readFile "../dist-newstyle/cache/plan.json"
```

Comment:
Instead of invoking `cardano-node` via `cabal` I now do it directly rather than go via `cabal exec`.  This is because I want to start the process in the test directory which is not possible via `cabal`.  In order to do that, I need to locate the executable which is done by inspecting the `plan.json` file.